src/main/java/org/jitsi/jigasi/xmpp/CallControlMucActivator.java

```diff
@@ -344,6 +344,10 @@ public void onSessionRemoved(AbstractGatewaySession session)
     public void onSessionFailed(AbstractGatewaySession session)
     {}
 
+    @Override
+    public void onReady()
```

Comment:
We already use java8 in jigasi, let's just get rid of all these dummy overrides and use default methods in the interface